### PR TITLE
Added information to card slots and vue components

### PR DESCRIPTION
**Pull request by LoganDemaray316**
Adds style and temporary information to cards that is the baseline for how students will view the dashboard.

### CODE REVIEW

File: src/components/EventComponent.vue [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<template>
+  <v-container fluid class="eventComponent">
+    <v-card class="eventsGradient fullBorderCurve">
```
Comment (ethanimooney) [inline src/components/EventComponent.vue:3]: Add mainBlur to the class here to give the card our computed drop shadow

File: src/components/EventComponent.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      <v-card class="ma-3 fullBorderCurve">
```
Comment (ethanimooney) [inline src/components/EventComponent.vue:4]: Give it elevation="0" so that this doesn't have a drop shadow

File: src/components/CritiqueComponent.vue [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            </v-avatar>
+          </v-col>
+          <v-col cols="11">
+            <v-card-title class="text-darkBlue font-weight-bold pb-2">
+              {{ jurors.at(2) }}
+            </v-card-title>
+          </v-col>
+        </v-row>
+      </v-card-text>
+      <v-card class="ma-5 outlined">
```
Comment (ethanimooney) [inline src/components/CritiqueComponent.vue:58]: Give this elevation="0" to get rid of the drop shadow

File: src/components/CritiqueComponent.vue [hunk 4]
```diff
@@ -0,0 +1,145 @@
+<template>
+  <v-container fluid class="eventComponent">
+    <v-card class="fullBorderCurve mainBlur">
+      <v-card-title class="font-weight-bold text-black">
+        <v-row>
+          <v-col>{{ title }}</v-col>
+          <v-col>
+            <v-btn class="outlined" rounded="pill" size="small">{{
```
Comment (ethanimooney) [inline src/components/CritiqueComponent.vue:8]: Give this elevation="0" to remove the drop shadow.

File: src/components/EventComponent.vue [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        <v-card-title class="font-weight-black text-darkGray">
+          <v-row>
+            <v-col>
+              {{ title }}
+            </v-col>
+            <v-col class="text-right">
+              <v-btn
```
Comment (ethanimooney) [inline src/components/EventComponent.vue:11]: Give this elevation="0" to remove the drop shadow.

File: src/components/StudentHomeDashboard.vue [hunk 6]
```diff
@@ -1,17 +1,82 @@
...
   <v-container fluid class="studentHomeMainGrid">
     <v-container fluid class="eventsRepPane studentHomeLeftGrid noPadMarg">
-      <v-card class="eventsPane mainBlur rounded-lg"> </v-card>
-      <v-card class="repertoirePane mainBlur rounded-lg"> </v-card>
+      <v-card class="eventsPane mainBlur rounded-lg">
+        <v-card-title class="font-weight-bold text-darkBlue">
+          <v-row>
+            <v-col> Your Events </v-col>
+            <v-col class="text-right">
+              <v-btn
```
Comment (ethanimooney) [inline src/components/StudentHomeDashboard.vue:9]: Give this elevation="0" to remove the drop shadow.

File: src/components/StudentHomeDashboard.vue [hunk 7]
```diff
@@ -1,17 +1,82 @@
...
+            </v-col>
+          </v-row>
+        </v-card-text>
+      </v-card>
+      <v-card class="repertoirePane mainBlur rounded-lg">
+        <v-card-title class="font-weight-bold text-darkBlue">
+          <v-row>
+            <v-col> Repertoire </v-col>
+            <v-col class="text-right">
+              <v-btn
```
Comment (ethanimooney) [inline src/components/StudentHomeDashboard.vue:34]: Give this elevation="0" to remove the drop shadow.

File: src/components/StudentHomeDashboard.vue [hunk 8]
```diff
@@ -1,17 +1,82 @@
...
+                size="small"
+                rounded="pill"
+                class="buttonGradient mr-3 text-white">
+                Add New
+              </v-btn>
+              <v-btn
```
Comment (ethanimooney) [inline src/components/StudentHomeDashboard.vue:40]: Give this elevation="0" to remove the drop shadow.

File: src/router/index.js [hunk 9]
```diff
@@ -9,11 +9,11 @@ const router = createRouter({
   history: createWebHistory(),
   routes: [
     {
-      path: "/dashboard",
+      path: "/",
       component: BaseDashboard,
       name: "baseDashboard",
     },
-    { path: "/", component: LoginPage, name: "loginPage" },
+    { path: "/login", component: LoginPage, name: "loginPage" },
```
Comment (ethanimooney) [inline src/router/index.js:16]: Undo this change, we don't want this pushed.

File: src/components/StudentHomeDashboard.vue (hunk 7 above)
Comment (ethanimooney) [reply src/components/StudentHomeDashboard.vue:34]: Go ahead and make all the buttons that we have font-weight-bold

File: src/components/CritiqueComponent.vue [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+              {{ jurors.at(2) }}
+            </v-card-title>
+          </v-col>
+        </v-row>
+      </v-card-text>
+      <v-card class="ma-5 outlined">
+        <v-card-title class="font-weight-bold text-darkBlue">
+          Musical Selection
+        </v-card-title>
+        <v-card-text>
```
Comment (ethanimooney) [inline src/components/CritiqueComponent.vue:62]: Can you try to get these right aligned with the title better?

File: src/components/RepertoireComponent.vue [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+            </v-card-subtitle>
+          </v-col>
+          <v-col cols="1" align-self="center" class="pl-0">
+            <v-btn variant="plain" size="x-large" class="pl-0">
+              <font-awesome-icon icon="fa-solid fa-video" class="text-white" />
+            </v-btn>
+          </v-col>
+          <v-col cols="1" align-self="center" class="pl-0">
+            <v-btn variant="plain" size="x-large" class="pl-0">
+              <font-awesome-icon
```
Comment (ethanimooney) [inline src/components/RepertoireComponent.vue:26]: You can change these sizes using the stuff built into vuetify for changing text size

**Review by ethanimooney — CHANGES_REQUESTED** [review]
In StudentHomeDashboard, on line 92 you can update `grid-template-columns: 3fr 2fr;` to `grid-template-columns: 3fr 1fr;` to shrink the critiques column.

File: src/components/EventComponent.vue (hunk 1 above)
Comment (LoganDemaray316) [reply src/components/EventComponent.vue:3]: Fixed

File: src/components/EventComponent.vue (hunk 2 above)
Comment (LoganDemaray316) [reply src/components/EventComponent.vue:4]: Fixed

File: src/components/CritiqueComponent.vue (hunk 3 above)
Comment (LoganDemaray316) [reply src/components/CritiqueComponent.vue:58]: Fixed

File: src/components/CritiqueComponent.vue (hunk 4 above)
Comment (LoganDemaray316) [reply src/components/CritiqueComponent.vue:8]: Fixed

File: src/components/EventComponent.vue (hunk 5 above)
Comment (LoganDemaray316) [reply src/components/EventComponent.vue:11]: Fixed

File: src/components/StudentHomeDashboard.vue (hunk 6 above)
Comment (LoganDemaray316) [reply src/components/StudentHomeDashboard.vue:9]: Fixed

File: src/components/StudentHomeDashboard.vue (hunk 8 above)
Comment (LoganDemaray316) [reply src/components/StudentHomeDashboard.vue:40]: Fixed

File: src/components/StudentHomeDashboard.vue (hunk 7 above)
Comment (LoganDemaray316) [reply src/components/StudentHomeDashboard.vue:34]: Both Fixed

File: src/components/CritiqueComponent.vue (hunk 10 above)
Comment (LoganDemaray316) [reply src/components/CritiqueComponent.vue:62]: Fixed

**Review by ethanimooney — APPROVED** [review]
Going to LGTM for now, can work on the icons later.

File: src/components/RepertoireComponent.vue (hunk 11 above)
Comment (LoganDemaray316) [reply src/components/RepertoireComponent.vue:26]: Fixed

File: src/router/index.js (hunk 9 above)
Comment (LoganDemaray316) [reply src/router/index.js:16]: Changed
